content/metadata.yaml

```diff
@@ -17,3 +17,11 @@ author_info:
       - School of Engineering and Applied Sciences, Harvard University
       - Department of Genetics, Harvard Medical School
       - Lab41, In-Q-Tel
+  -
+    github: AlexanderTitus
```

Comment:
@Benjamin-Lee do you want contributors/authors to start adding themselves to `metadata.yaml` now or after the ordering has been determined?  The order in this file will be the order in the generated HTML and PDF.

Reply:
To follow up, I'm not questioning whether @AlexanderTitus should be an author.  His contributions to the project are quite clear.  The question was only about the timing of listing authors.